books/free-programming-books-langs.md

```diff
@@ -536,9 +536,9 @@ Books on general-purpose programming that don't focus on a specific language are
 * [Joint Strike Fighter, C++ Coding Standards](http://www.stroustrup.com/JSF-AV-rules.pdf) - Bjarne Stroustrup (PDF)
 * [LearnCpp.com](https://www.learncpp.com) (HTML)
 * [Learning C++ eBook](https://riptutorial.com/Download/cplusplus.pdf) - Compiled from StackOverflow Documentation (PDF)
-* [Lecture Notes On Object Oriented Programming Using C++](https://www.cet.edu.in/noticefiles/285_OOPS%20lecture%20notes%20Complete.pdf) - Dr. Subasish Mohapatra (PDF)
 * [Matters Computational: Ideas, Algorithms, Source Code](http://www.jjj.de/fxt/fxtbook.pdf) - Jorg Arndt (PDF)
 * [More C++ Idioms](https://en.wikibooks.org/wiki/More_C%2B%2B_Idioms) - Sumant Tambe, et al. (WikiBooks)
+* [Object Oriented Programming using 'C++'](https://www.ddegjust.ac.in/studymaterial/msc-cs/ms-17.pdf) - Pooja Chawla (PDF)
```

Comment:
The link you provided isn't a book; it appears to be a study material for a university course.